StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1513ClosingCurlyBracketMustBeFollowedByBlankLine.cs

```diff
@@ -249,6 +249,13 @@ private void AnalyzeCloseBrace(SyntaxToken token)
                         return;
                     }
 
+                    if (nextToken.IsKind(SyntaxKind.ColonToken)
+                        && IsPartOf<ConditionalExpressionSyntax>(token))
```

Comment:
:memo: I simplified this condition to just the following during the merge:

```csharp
if (nextToken.IsKind(SyntaxKind.ColonToken))
```

Even if there was another case, whatever that case is would also be exempt from this rule.